cmake/systemd.cmake

```diff
@@ -0,0 +1,56 @@
+INCLUDE(FindPkgConfig)
```

Comment:
It would be nice if you could add copyright header to this file. And also to my_systemd.h.